frontend/src/components/wishlist/wishlist.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  ComplectationResponseDto,
+  DeleteWishlistInput,
+  ModelResponseDto,
+  WishlistInput,
+} from '@autoline/shared/common/types/types';
+import { Title } from '@components/common/title/title';
+import { NewCarCard } from '@components/new-car-card/new-car-card';
+import { Grid } from '@mui/material';
+import {
+  useGetWishlistsQuery,
+  useCreateWishlistMutation,
+  useDeleteWishlistMutation,
+} from '@store/queries/preferences/wishlist';
+
+const Wishlist: React.FC = () => {
+  const { data, isLoading } = useGetWishlistsQuery();
+
+  const [createWishlist] = useCreateWishlistMutation();
+  const [deleteWishlist] = useDeleteWishlistMutation();
+
+  const handleCreate = async (args: WishlistInput): Promise<void> => {
+    await createWishlist(args);
+  };
+
+  const handleDelete = async (args: DeleteWishlistInput): Promise<void> => {
+    await deleteWishlist(args);
+  };
+
+  return (
+    <>
+      {!isLoading && (
+        <>
```

Comment:
This can be simplified as:
```suggestion
  if (isLoading) return null;
  return (
    <>
```